decidim-verifications/app/events/decidim/verifications/managed_user_error_event.rb

```diff
@@ -26,7 +25,13 @@ def resource_title
       end
 
       def default_i18n_options
-        super.merge({ conflicts_path: conflicts_path, managed_user_path: managed_user.profile_path, managed_user_name: managed_user.name })
+        super.merge({
+                      conflicts_path: conflicts_path,
+                      conflicts_url: conflicts_url(host: managed_user.organization.host),
```

Comment:
With these URLs we have the same problem at development environment as #9154. And with production configuration, I believe this is pointing to the `http` protocol instead of `https`, although not that big deal if redirects are handled properly (as they should).

When implementing the fixes and refactoring for that, I noticed that we can use the `Decidim::EngineRouter` to provide all the necessary options for these URLs. Maybe we could utilize that here as well?

The following code would provide the correct URL in this particular scenario also at the development environment:
```ruby
Decidim::EngineRouter.new("decidim_admin", { host: managed_user.organization.host }).conflicts_url
```

I would maybe wrap the engine router generation to its own method and then call the URL helpers through that.